config/locales/en.yml

```diff
@@ -21,3 +21,9 @@
 
 en:
   hello: "Hello world"
+  votes: {
```

Comment:
ah.  yaml doesn't like ,s apparently.  one of those whitespace things.  

thanks richard for the help